Cargo.toml

```diff
@@ -11,7 +11,7 @@ kelvin = "0.18"
 nstack = "0.4"
 lazy_static = "1.3.0"
 hades252 = { git = "https://github.com/dusk-network/hades252", tag = "v0.7.0" }
-dusk-plonk = "0.2.7"
+dusk-plonk = "0.2.8"
```

Comment:
Couldn't we set `dusk-plonk = "0.2"` so it'll always take the latest one avobe?

Reply:
I think its not desirable considering plonk is `MAJOR=0`, so the patch is in practice the minor version.

If we go to version `1` of plonk, then that will be fine. Before that, any minor version upgrade will be enforced and these sometimes can be breaking

Reply:
Makes sense. Bump up the version of Poseidon anyway.